docs/getting-started/concepts/design-principles.md

```diff
@@ -0,0 +1,19 @@
+# Design Principles
+
+Bruin is an opinionated data tool, and in some sense it can be considered as a framework to build scalable data products, both in terms of data size as well as the _data team size_. Below is a living list of design principles that outlines the vision behind Bruin, both as an open-source product as well as a business.
+
+1. **Everything must be done via some form of a version-controllable text.** This means there should be no UI/database to configure anything in terms of how Bruin pipelines run.
+2. **You need multiple technologies.** While SQL is incredibly powerful, there are a lot of usecases that are not doable via just SQL. Bruin supports both Python and SQL natively, as well as pre-built images/binaries to support more complex usecases.
+3. **You need multiple sources and destinations.** Any company above a certain size would use more than one platform that has relevant data. This can be marketing platforms, CRM systems, data warehouses, data lakes, whatever. Data pipelines must be built to support the diverse set of sources and destinations.
```

Comment:
```suggestion
3. **You need multiple sources and destinations.** Any company above a certain size would use more than one platform that has relevant data. This can be marketing platforms, CRM systems, data warehouses, data lakes and many more. Data pipelines must be built to support the diverse set of sources and destinations.
```